src/structures/cl_args.cpp

```diff
@@ -26,13 +27,29 @@ void update_host(Servers& servers, const std::string& value) {
     host = value.substr(index + 1);
   }
 
+  if (!host.empty()) {
+    // remove any trailing slash
+    if (host.back() == '/') {
+      host.pop_back();
+    }
+
+    // pull out a path if any and append a trailing slash for query building
+    index = host.find('/');
+    if (index != std::string::npos) {
+      path = host.substr(index + 1) + "/";
+      host = host.substr(0, index);
```

Comment:
Nitpicking: this is making a copy while we could simply `host.erase(index)`.

Reply:
good point! done. haha but too late. feel free to change it before next release